<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>404</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>登陆</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>54</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>VIP用户</string>
    </attribute>
    <widget class="QPushButton" name="bt_vip_login">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登 录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_register">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>注 册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_user_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>53</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>经 理</string>
    </attribute>
    <widget class="QPushButton" name="bt_mana_login">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登 录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_user_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>53</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_user_login">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>普通用户登录</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
